aspnetcore/signalr/groups.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Manage users and groups in SignalR
+author: 
+description: 
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: 
+ms.custom: mvc
+ms.date: 06/04/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/groups
+---
+
+# Manage users and groups in SignalR
```

Comment:
nit: Manag**ing**

Reply:
Can't use Managing. It's a gerund and docs guidelines say no gerunds.

Reply:
I sit corrected 😄